static/js/src/advantage/react/components/Subscriptions/SubscriptionDetails/DetailsContent/DetailsContent.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, CodeSnippet, Col, Row } from "@canonical/react-components";
+import { CodeSnippetBlockAppearance } from "@canonical/react-components/dist/components/CodeSnippet";
+import React from "react";
+
+import DetailsTabs from "../DetailsTabs";
+
+type Feature = {
+  size?: number;
+  title: string;
+  value: string | number;
+};
+
+const generateFeatures = (features: Feature[]) =>
+  features.map(({ size = 3, title, value }) => (
+    <Col key={title} size={size}>
```

Comment:
Type error on the size

Reply:
Thanks, I rebased on master to get the new react-components so this type changed and because the linter is broken I didn't spot that.